designer/server/src/main/scala/pl/touk/nussknacker/ui/api/description/scenarioActivity/Dtos.scala

```diff
@@ -101,105 +102,107 @@ object Dtos {
 
   object ScenarioActivityType extends Enum[ScenarioActivityType] {
 
+    private val commentRelatedActions = List("delete_comment", "edit_comment")
+
     case object ScenarioCreated extends ScenarioActivityType {
       override def displayableName: String        = "Scenario created"
```

Comment:
Could you elaborate on what I should do regarding these things :) ?

Reply:
Ok, I think I understand the case of the icons, as I see they are not available now, so I need to export them from Figma add them to our codebase, and pass the correct path to based on the activity type. What about displayableName?

Reply:
About the icons - precisely that, the metadata endpoint will return info about icon for activity type. 
About the displayable name - it is the name, that should be displayed as "title"/"header" of the activity on the list. It is returned in metadata to avoid a lot of duplication in JSONs. The current values are taken from Figma, maybe something will need adjusting when working on FE, maybe not.